core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -73,6 +73,28 @@ coreos:
 
         [Install]
         RequiredBy=format-etcd2-volume.service
+{{if and (.AmazonSsmAgent.Enabled) (ne .AmazonSsmAgent.DownloadUrl "")}}
+    - name: amazon-ssm-agent.service
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=amazon-ssm-agent
+        Requires=network-online.target
+        After=network-online.target
+
+        [Service]
+        Type=simple
+        WorkingDirectory=/
```

Comment:
Seems so according to https://www.freedesktop.org/software/systemd/man/systemd.exec.html#WorkingDirectory=